shell/artifacts/hex/join-game.js

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) 2018 Google Inc. All rights reserved.
+// This code may only be used under the BSD style license found at
+// http://polymer.github.io/LICENSE.txt
+// Code distributed by Google as part of this project is also
+// subject to an additional IP rights grant found at
+// http://polymer.github.io/PATENTS.txt
+
+defineParticle(({SimpleParticle}) => {
+  return class extends SimpleParticle {
+    constructor() {
```

Comment:
Do we need to do this (here and next particle)? Asking as I don't think I've done.

Reply:
Not needed. Removed.